src/Neo.CLI/config.fs.mainnet.json

```diff
@@ -30,8 +30,8 @@
   "ProtocolConfiguration": {
     "Network": 91414437,
     "AddressVersion": 53,
-    "MillisecondsPerBlock": 15000,
-    "MaxTransactionsPerBlock": 512,
+    "MillisecondsPerBlock": 3000,
```

Comment:
Please don't change FS networks, we will use 1s there, but not right now.

Reply:
Ok, can you revert ?